x/merkledb/proof.go

```diff
@@ -98,16 +95,16 @@ func (node *ProofNode) UnmarshalProto(pbNode *pb.ProofNode, bf BranchFactor) err
 	case pbNode.Key == nil:
 		return ErrNilKey
 	}
-	node.Key = ToKey(pbNode.Key.Value, bf).Take(int(pbNode.Key.Length))
+	node.Key = ToKey(pbNode.Key.Value).Take(int(pbNode.Key.Length))
```

Comment:
Not new as part of this PR, but couldn't someone craft a `pb.ProofNode` to cause a panic here?

Reply:
no